src/test/java/bio/terra/service/journal/JournalServiceTest.java

```diff
@@ -201,29 +219,19 @@ private void validateEntries(
       String note,
       Map<String, Object> entryMap) {
     List<JournalEntryModel> results = journalService.getJournalEntries(key, resourceType, 0, 100);
-    JournalEntryModel entryUnderTest = results.get(expectedCount - 1);
     assertThat(
-        "the number of entries matches the expected count.",
-        results.size(),
-        equalTo(expectedCount));
+        "the number of entries matches the expected count.", results, hasSize(expectedCount));
+    JournalEntryModel entryUnderTest = results.get(0);
     assertThat(
         "the journal resource key matches the UUID", entryUnderTest.getResourceKey(), equalTo(key));
     assertThat(
         "the journal entry resource type is the expected value",
-        entryUnderTest.getResourceType().toString().toLowerCase(),
```

Comment:
I consolidated some of these assertions. They seemed redundant to me to check both upper and lower cased equality.